src/test/java/LoginTests.java

```diff
@@ -4,19 +4,18 @@ public class LoginTests extends BaseTest {
 
 
     @Test
-    public void loginValidEmailPasswordTest() {
+    public void LoginEmptyEmailPasswordTest() {
+        openLoginURL();
+        provideEmail("guadalupe.medina@testpro.io");
```

Comment:
Please, add more test 

HINT

   **Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**